tests/monitoring/monitoring_service/test_handlers.py

```diff
@@ -440,6 +441,24 @@ def test_monitor_new_balance_proof_event_handler_idempotency(context: Context,):
     assert channel.update_status.update_sender_address == "C"
 
 
+def test_monitor_reward_claimed_event_handler(context: Context, log):
+    context = setup_state_with_closed_channel(context)
+
+    claim_event = ReceiveMonitoringRewardClaimedEvent(
+        ms_address=context.ms_state.address,
+        amount=TokenAmount(1),
+        reward_identifier="REWARD",
+        block_number=BlockNumber(23),
+    )
+
+    monitor_reward_claim_event_handler(claim_event, context)
+    assert log.has("Successfully claimed reward")
+
+    claim_event.ms_address = Address("C")
+    monitor_reward_claim_event_handler(claim_event, context)
+    assert log.has("Another MS claimed reward event")
+
```

Comment:
Do we need to include a test or a log when the MS claims with the wrong token amount?

Reply:
This is just the event, the transaction itself is sent in `action_claim_reward_triggered_event_handler`